### PR TITLE
feat(FileUpload): remove onChange prop

### LINKED ISSUE #135: FileUpload: removed deprecated onChange prop
Original issue: https://github.com/patternfly/patternfly-react/pull/8155

Description:  We've removed the `onChange` prop completely with no replacement, should remove it for consumers and alert them to this change.

_Required actions:_
1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by thatblindgeye**
Closes #135

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/test/rules/v5/fileUpload-remove-onChange.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/fileUpload-remove-onChange");
+
+ruleTester.run("fileUpload-remove-onChange", rule, {
+  valid: [
+    {
+      code: `import { FileUpload } from '@patternfly/react-core'; <FileUpload />`,
+    },
+    // No @patternfly/react-core import
+    { code: "<FileUpload />" },
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/fileUpload-remove-onChange.js:10]: This should include the attribute, the reason for these tests is that we won't touch it if a user has a FileUpload that is not from PF.

**Review by gitdallas — CHANGES_REQUESTED** [review]
small change

File: packages/eslint-plugin-pf-codemods/test/rules/v5/fileUpload-remove-onChange.js (hunk 1 above)
Comment (gitdallas) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/fileUpload-remove-onChange.js:10]: @kmcfaul  - I just noticed the same thing for your visiblity test, although the chances of another `ToolbarContent` with a wrongly spelled "visiblity" attribute is... pretty small

Comment (thatblindgeye) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/fileUpload-remove-onChange.js]: Thanks for catching this!

**Review by gitdallas — APPROVED** [review]

**Pull request merged**